RTATimer/RTATimer.lua

```diff
@@ -0,0 +1,16 @@
+ModUtil.RegisterMod("RTATimer")
+
+local config = {
+  ModName = "Timer",
+  UseRealTime = false
+}
+
+if ModConfigMenu then
+  ModConfigMenu.Register(config)
+end
+
+ModUtil.WrapBaseFunction("AddTimerBlock", function(baseFunc, ...)
+  if config.UseRealTime then return end
+  return baseFunc(...)
+end, RTATimer)
```

Comment:
I know this adds complexity, but ideally we would support having both timers running at the same time.  Happy to start here as a toggle which gets most of the value tho